src/controllers/expenses.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const expensesServices = require('../services/expenses.js');
+const userServices = require('../services/users');
+
+class ExpenseController {
+  getAll(req, res) {
+    const { userId, category, from, to } = req.query;
+
+    let vizibleExpenses = expensesServices.getAll();
+
+    if (userId) {
+      vizibleExpenses = vizibleExpenses
+        .filter(expense => expense.userId === +userId);
+    };
+
+    if (category) {
+      vizibleExpenses = vizibleExpenses
+        .filter(expense => expense.category === category);
+    }
+
+    if (from) {
+      vizibleExpenses = vizibleExpenses
+        .filter(expense => expense.spentAt >= from && expense.spentAt <= to);
+    }
```

Comment:
Maybe code would be more elegant in you would split this logic into 3 different methods. What do you think?